File: core/src/com/mygdx/game/GdxGame.java
package com.mygdx.game;

import com.badlogic.gdx.Game;
import com.mygdx.game.screens.GameScreen;
import com.mygdx.game.screens.StartScreen;

public class GdxGame extends Game {

	@Override
	public void create () {
		//setScreen(new GameScreen());
		setScreen(new StartScreen());
	}

	@Override
	public void render () {
		super.render();
	}
	
	@Override
	public void dispose () {
	}
}

File: core/src/com/mygdx/game/GdxGame.kt
package com.mygdx.game

import com.badlogic.gdx.Game
import com.mygdx.game.managers.ScreenManager
import com.mygdx.game.managers.ScreenManager.Param.SERVICES_CONTROLLER
import com.mygdx.game.managers.ScreenManager.Screens.*
import com.mygdx.game.services.ServicesController

class GdxGame(private val controller: ServicesController) : Game() {

    override fun create() {
        ScreenManager.game = this
        ScreenManager.globalParameters[SERVICES_CONTROLLER] = controller
        ScreenManager.setScreen(LOADING_SCREEN)
        controller.signIn()
    }

    override fun render() {
        super.render()
    }

    override fun dispose() {}
}
